dlp/snippets/deid.py

```diff
@@ -1125,15 +1125,17 @@ def deidentify_with_exception_list(
 
 
 # [START dlp_deidentify_table_bucketing]
+from typing import Dict  # noqa: F811, E402, I100
 from typing import List  # noqa: F811, E402, I100
+from typing import Union  # noqa: F811, E402, I100
```

Comment:
nit: consider putting this in a single import, i.e. `from typing import Dict, List, Union`

Reply:
updated